centraldogma/src/main/java/com/linecorp/decaton/centraldogma/CentralDogmaPropertySupplier.java

```diff
@@ -225,12 +252,24 @@ private static long remainingTime(long totalTime, long startedTime) {
     }
 
     // visible for testing
-    static JsonNode defaultProperties() {
-        final ObjectNode properties = objectMapper.createObjectNode();
+    static JsonNode convertPropertyListToJsonNode(List<Property<?>> properties) {
+        final ObjectNode propertiesObjectNode = objectMapper.createObjectNode();
+        properties.forEach(
+                property -> {
+                    propertiesObjectNode.set(
+                            property.definition().name(),
+                            objectMapper.valueToTree(property.value())
+                    );
+                }
+        );
+        return propertiesObjectNode;
+    }
+
+    // visible for testing
+    static List<Property<?>> defaultProperties() {
```

Comment:
I think this functionality can be required by various supplier implementations? Maybe it's worth putting this method in `ProcessorProperties` rather than here privately.

Reply:
I moved this method `defaultProperties`, which returns `List<Property<?>>`,  to `ProcessorProperties`. https://github.com/line/decaton/pull/134/commits/03ef312262463e153abaf00dbd83410702f4de2b